### PR TITLE
feat: Add custom cluster selector tutorial

### PR DESCRIPTION
**Pull request by Akash-Nayak**
Signed-off-by: Akash Nayak <akash19nayak@gmail.com>

### CODE REVIEW

File: docs/tutorials/custom-cluster-selector.md [hunk 1]
```diff
@@ -0,0 +1,637 @@
+---
+layout: default
+title: "Customizing Kubernetes YAMLs to target specific cluster"
+permalink: /tutorials/customize-cluster-selector
+parent: Tutorials
+nav_order: 9
```
Comment (ashokponkumar) [inline docs/tutorials/custom-cluster-selector.md:6]: Also change nav_order for below tutorials to 91 and 92.

File: docs/tutorials/custom-cluster-selector.md [hunk 2]
```diff
@@ -0,0 +1,637 @@
...
+
+1. Install the [Move2Kube CLI tool](https://move2kube.konveyor.io/installation/cli)
+
+1. Install [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)
+
+1. We will use [kubernetes-to-kubernetes](https://github.com/konveyor/move2kube-demos/tree/main/samples/kubernetes-to-kubernetes) sample. The `kubernetes-to-kubernetes` directory has some Kubernetes YAMLs that deploy a web app with multiple services. There are 3 services: a frontend website in PHP, a backend API in NodeJS and a cache service using Redis.
+
+## Steps
+
+1. Let's first collect the data about your Kubernetes cluster using `move2kube collect`. Here we will limit the collect to only cluster information using the `-a k8s` annotation flag. Please note: Before running the below command, log in to your target cluster.
```
Comment (ashokponkumar) [inline docs/tutorials/custom-cluster-selector.md:25]: We need to mention about the need to have the cluster in context for kubectl

**Review by ashokponkumar — CHANGES_REQUESTED** [review]
Nice! couple of suggestions

File: docs/tutorials/custom-cluster-selector.md (hunk 1 above)
Comment (Akash-Nayak) [reply docs/tutorials/custom-cluster-selector.md:6]: Done

File: docs/tutorials/custom-cluster-selector.md (hunk 2 above)
Comment (Akash-Nayak) [reply docs/tutorials/custom-cluster-selector.md]: Done.

**Pull request merged**